ipaserver/install/httpinstance.py

```diff
@@ -382,18 +385,20 @@ def __setup_ssl(self):
             if len(server_certs) == 0:
                 raise RuntimeError("Could not find a suitable server cert in import in %s" % self.pkcs12_info[0])
 
+            # this is CA-less and we don't know the server-cert nickname
+            if self.cert_nickname is None:
+                # We only handle one server cert
+                self.cert_nickname = server_certs[0][0]
             self.create_password_conf()
 
-            # We only handle one server cert
-            nickname = server_certs[0][0]
-            if nickname == 'ipaCert':
-                nickname = server_certs[1][0]
```

Comment:
Removing this is not safe, there are various reasons `ipaCert` might be present in /etc/httpd/alias even now (previous install of older IPA version, etc.)